autokeras/tasks/image.py

```diff
@@ -221,3 +221,107 @@ def fit(self,
                     validation_split=validation_split,
                     validation_data=validation_data,
                     **kwargs)
+
+
+class ImageSegmenter(SupervisedImagePipeline):
+    """AutoKeras image segmentation class.
+
+    # Arguments
+        num_classes: Int. Defaults to None. If None, it will infer from the data.
```

Comment:
"it will be inferred from" (you can also apply this fix to the other docstrings)